spiceaidocs/docs/local-acceleration/index.md

```diff
@@ -29,7 +29,11 @@ Dataset acceleration can be configured in `full` (the entire dataset is refreshe
 
 ## Refresh Interval
 
-The data refresh interval for an accelerated dataset in `full` mode can be specified via [refresh_interval](/reference/spicepod/datasets#accelerationrefresh_interval) parameter (see Example section below), triggered via [refresh](cli/reference/refresh) command or API (`POST`, `/v1/datasets/:name/refresh`).
+The data refresh interval for an accelerated dataset in `full` mode can be specified via [`refresh_interval`](/reference/spicepod/datasets#accelerationrefresh_interval) parameter (see Example section below), triggered via [refresh](cli/reference/refresh) command or API (`POST`, `/v1/datasets/:name/refresh`).
```

Comment:
This describes how it can be specified but doesn't actually describe what the Refresh Interval is. We should describe specifically and succinctly what it is, and then how to set it.